renku/service/controllers/remote_project.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2020 - Swiss Data Science Center (SDSC)
+# A partnership between École Polytechnique Fédérale de Lausanne (EPFL) and
+# Eidgenössische Technische Hochschule Zürich (ETHZ).
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Renku service parent remote controller."""
```

Comment:
```suggestion
"""Renku service parent controller for remote support."""
```

Reply:
Actually I just had a realization, thanks for pointing this out. This module `remote_project` is actually not a controller, but a utility which is used by the controller. I renamed this to `utils.py` under `controllers` module.

New description is:
```
"""Utilities for renku service controllers."""
```